modules/data-entry/src/main/frontend/src/questionnaire/TimeQuestion.jsx

```diff
@@ -58,9 +58,23 @@ import DateTimeUtilities from "../components/DateTimeUtilities";
 function TimeQuestion(props) {
   checkPropTypes(TimeQuestion, props);
   let {existingAnswer, classes, pageActive, ...rest} = props;
-  let {text, lowerLimit, upperLimit, errorText, minAnswers, dateFormat} = {dateFormat: "HH:mm", ...props.questionDefinition, ...props};
-  let currentStartValue = (existingAnswer && existingAnswer[1].value && DateTime.fromFormat(existingAnswer[1].value, dateFormat).isValid)
-    ? DateTime.fromFormat(existingAnswer[1].value, dateFormat) : null;
+  let {
+    text,
+    lowerLimit,
+    upperLimit,
+    errorText,
+    minAnswers,
+    dateFormat,
+    saveFormat
+  } = {
+    dateFormat: "HH:mm",
+    saveFormat: "HH:mm:ss.SSS",
+    ...props.questionDefinition,
+    ...props
+  };
+  let currentStartValue = (existingAnswer && existingAnswer[1].value && DateTime.fromFormat(existingAnswer[1].value, saveFormat).isValid)
```

Comment:
Can we take this opportunity to make the statement shorter?

```suggestion
  let currentStartValue = (existingAnswer?.[1]?.value && DateTime.fromFormat(existingAnswer[1].value, saveFormat).isValid)
```